cypress/fixtures/mock-country.json

```diff
@@ -0,0 +1,106 @@
+{
+  "country": {
+      "data": {
+          "id": "3",
+          "type": "country",
+          "attributes": {
+              "name": "Brazil",
+              "description": "Pão de queijo is a famous Brazilian cheese bread made with cassava flour and cheese, resulting in a chewy and cheesy delight."
+          },
+          "relationships": {
+              "breads": {
+                  "data": [
+                      {
+                          "id": "13",
+                          "type": "bread"
+                      },
+                      {
+                          "id": "14",
+                          "type": "bread"
+                      },
+                      {
+                          "id": "15",
+                          "type": "bread"
+                      },
+                      {
+                          "id": "16",
+                          "type": "bread"
+                      }
+                  ]
+              }
+          }
+      }
+  },
+  "breads": {
+      "data": [
+          {
+              "id": "13",
+              "type": "bread",
+              "attributes": {
+                  "name": "Pão de Queijo",
+                  "description": "Pão de queijo is a famous Brazilian cheese bread made with cassava flour and cheese, resulting in a chewy and cheesy delight.",
+                  "recipe": "Ingredients: 500g tapioca flour 250ml milk 125g butter 10g salt 2 eggs 200g grated cheese Instructions: Boil milk with butter and salt. Pour over tapioca flour and mix. Let it cool, then add eggs and cheese. Form small balls and bake at 375°F (190°C) for 15-20 minutes."
```

Comment:
This will change once we shift over to the NEW AND IMPROVED back end, but just commenting to try and triangulate what I know in here.